build.sbt

```diff
@@ -4,16 +4,18 @@ import play.core.PlayVersion.current
 lazy val scoverageSettings = {
   import scoverage.ScoverageKeys
   Seq(
+    //    ScoverageKeys.coverageExcludedPackages := "<empty>;Reverse.*;uk.gov.hmrc.BuildInfo;app.*;prod.*;config.*;com.*;.*models.*",
```

Comment:
commented code.

Reply:
how did I miss that :(  (was seeing if a shorter list helped )